lazy-lock.json

```diff
@@ -1,47 +1,62 @@
 {
-  "Comment.nvim": { "branch": "master", "commit": "e89df176e8b38e931b7e71a470f923a317976d86" },
-  "LuaSnip": { "branch": "master", "commit": "53e812a6f51c9d567c98215733100f0169bcc20a" },
-  "bufferline.nvim": { "branch": "main", "commit": "2d5266d14b7b5dbc54923469b4599f0349f7cedc" },
+  "Comment.nvim": { "branch": "master", "commit": "0236521ea582747b58869cb72f70ccfa967d2e89" },
```

Comment:
Should this file actually be part of the repository? It is dependent on each user configuration and does not serve any purpose because we can pin the packages version by ourselves.

Reply:
Theoretically yes, the lazy lockfile allows us to run the doom installation on multiple machines and ensure that they all have the same initial plugins _and also the same version of said plugins before the user updates them themselves if necessary_. Take a look at the [lockfile](https://github.com/folke/lazy.nvim?tab=readme-ov-file#-lockfile-lazy-lockjson) section in the lazy.nvim documentation :p